src/deepmol/models/keras_models.py

```diff
@@ -137,15 +142,47 @@ def fit_on_batch(self, X: Sequence, y: Sequence):
         Fits model on batch of data.
         """
 
-    def reload(self) -> None:
+    @classmethod
+    def load(cls, file_path: str, **kwargs) -> 'KerasModel':
```

Comment:
Just wondering if there is no way of getting these kwargs directly from the saved model. Otherwise, the user must know and provide them every time he loads a model.